Add alternative script location for brew installs (#104)
halcyon/asdf-java: README.md

For fish shell, instead use:

`. ~/.asdf/plugins/java/set-java-home.fish`

Note that the location of the scripts will be different if you installed asdf with a package manager.

> For example, you you have installed asdf using brew, the scripts will be located in

Reviewer: ```suggestion
For example, if you have installed asdf using Homebrew, the scripts will be located in
```

> `/usr/local/opt/asdf/plugins/java/`

Reviewer: Should we use the output of `brew --prefix asdf` instead? The path is different if Homebrew was installed in a different location:
https://github.com/Homebrew/brew/blob/2100335441a7068abb5a08be1e01086b44617ae5/docs/Installation.md#alternative-installs

```suggestion
`$(brew --prefix asdf)/plugins/java/`
```

Author: Good call!

The commands must be altered accordingly, e.g.

> `. /usr/local/opt/asdf/plugins/java/set-java-home.zsh`

Reviewer: ```suggestion
`. $(brew --prefix asdf)/plugins/java/set-java-home.zsh`
```

> `. $(brew --prefix asdf)/plugins/java/set-java-home.zsh`

Author: ```suggestion
`. "$(brew --prefix asdf)/plugins/java/set-java-home.zsh"`
```

And we need to add quotes in case there is a space in the custom install location
